atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -199,6 +200,15 @@ private boolean isPrimaryKeyViolation(PalantirSqlException ex) {
 
     @Override
     public void drop() {
+        drop(true);
+    }
+
+    /**
+     * Drops the table, and deletes the table from the mapping table (if present), and from the metadata table.
+     * If treatReferenceAsCaseSensitive is false, and another table is created with a table reference that is a case
+     * insensitive match with this table, then the behaviour is undefined.
+     */
+    public void drop(boolean treatReferenceAsCaseSensitive) {
```

Comment:
I'd suggest using an enum since this is new code: maybe `CaseSensitivity referenceCaseSensitivity`